types/diff-match-patch/index.d.ts

```diff
@@ -40,13 +44,22 @@ export declare class diff_match_patch {
     diff_toDelta(diffs: Diff[]): string;
     diff_fromDelta(text1: string, delta: string): Diff[];
 
-    patch_make(text1: any, text2?: string): Patch[];
-    patch_deepCopy(patches: Patch[]): Patch[];
-    patch_apply(patches: Patch[], text: string): [string, boolean[]];
-    patch_fromText(text: string): Patch[];
-    patch_toText(patches: Patch[]): string;
+	match_main(text: string, pattern: string, loc: number): number;
+    match_bitap_(text: string, pattern: string, loc: number): number;
+    match_alphabet_(pattern: string): {[char: string]: number};
+
+    patch_addContext_(patch: patch_obj, text: string): void;
+    patch_make(a: string, opt_b: string | Diff[]): patch_obj[];
+    patch_make(a: Diff[]): patch_obj[];
+	patch_make(a: string, opt_b: string, opt_c: Diff[]): patch_obj[];
```

Comment:
You might just wanna grep the file for tab characters and replace them with spaces, because there's still tabs in this diff.